Gordon360/Models/CCT/Context/ICCTContextProcedures.cs

```diff
@@ -45,6 +45,7 @@ public partial interface ICCTContextProcedures
         Task<int> DINING_INFO_BY_STUDENT_IDAsync(int? STUDENT_ID, string SESS_CDE, OutputParameter<int> returnValue = null, CancellationToken cancellationToken = default);
         Task<List<DISTINCT_ACT_TYPEResult>> DISTINCT_ACT_TYPEAsync(string SESS_CDE, OutputParameter<int> returnValue = null, CancellationToken cancellationToken = default);
         Task<List<EMAILS_PER_ACT_CDEResult>> EMAILS_PER_ACT_CDEAsync(string ACT_CDE, string SESS_CDE, OutputParameter<int> returnValue = null, CancellationToken cancellationToken = default);
+        Task<List<EVENTS_BY_STUDENT_IDResult>> EVENTS_BY_STUDENT_IDAsync(string STU_USERNAME, OutputParameter<int> returnValue = null, CancellationToken cancellationToken = default);
```

Comment:
Did you hit the checkbox to include all Stored Procedures? This SP should not be included.

Reply:
I have this in my current version, I don't know what happened

Reply:
I think you may have misunderstood my question - this Stored Procedure got **added** when you refreshed from the database, which we do not want to happen. You can choose which stored procedures to include or not in the auto-generated mappings. I'm wondering if you somehow checked this to get included?

Reply:
Ohhhh, whoops. Misread, I think I might have added the SP's